cdc/generation.cc

```diff
@@ -25,6 +25,7 @@
 #include "gms/inet_address.hh"
```

Comment:
Re Commit message: please describe how the test fails without the changes from this PR (i.e. if we try to send the data in a single command).

Reply:
Done.